.github/actions/mike-test-build/action.yml

```diff
@@ -0,0 +1,60 @@
+name: Publish Artifacts
```

Comment:
Probably we should remove the test actions?

Reply:
I guess I wasn't sure what the aim was for this action. I figured you're trying to potentially get the builds running for the temp branch? The trick is the docker run step down below. I'll have to replace the other action if we want to get rid of it.